vortex-serde/src/file/reader/mod.rs

```diff
@@ -14,25 +15,29 @@ use vortex::compute::unary::subtract_scalar;
 use vortex::compute::{and, filter, search_sorted, slice, take, SearchSortedSide};
 use vortex::{Array, ArrayDType, IntoArray, IntoArrayVariant};
 use vortex_dtype::{match_each_integer_ptype, DType, StructDType};
-use vortex_error::{vortex_bail, VortexError, VortexResult};
+use vortex_error::{vortex_bail, VortexResult};
 use vortex_scalar::Scalar;
 
 use super::layouts::{Layout, StructLayout};
 use crate::file::file_writer::MAGIC_BYTES;
 use crate::file::footer::Footer;
 use crate::io::VortexReadAt;
-use crate::{ArrayBufferReader, ReadResult};
 
+mod batch;
+mod column;
 pub mod filtering;
 pub mod projections;
 pub mod schema;
 
+const DEFAULT_BATCH_SIZE: usize = 1024;
```

Comment:
This probably needs to be bigger, 
```suggestion
const DEFAULT_BATCH_SIZE: usize = 65536;
```